projects/packages/videopress/src/client/block-editor/blocks/video/index.native.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { getJetpackExtensionAvailability } from '@automattic/jetpack-shared-extension-utils';
+import { registerBlockType } from '@wordpress/blocks';
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import { VideoPressIcon as icon } from './components/icons';
+import Edit from './edit';
+import save from './save';
+import transforms from './transforms';
+import videoPressBlockExampleImage from './videopress-block-example-image.jpg';
+import './style.scss';
+
+export const { name, title, description, attributes } = metadata;
+
+/**
+ * Registers the VideoPress block if the availability requirements are met.
+ *
+ * @returns {object|boolean} Either false if the block is not available, or the results of `registerBlockType`
+ */
+export default function registerVideoPressBlock() {
+	const { available } = getJetpackExtensionAvailability( name );
```

Comment:
I followed the same pattern that we're using to check for the availability of other Jetpack blocks on mobile, as seen in [the `register-jetpack-block.native.js` file](https://github.com/Automattic/jetpack/blob/732bf89cdb4fca04780b64922987c42d7fefc002/projects/plugins/jetpack/extensions/shared/register-jetpack-block.native.js).